src/core/importAtom.nix

```diff
@@ -89,7 +89,46 @@ let
       ) config.fetch or { }
     # else if fetcher = "native", etc
     else
-      { };
+      let
+        lockPath = "${root}/${src}.lock";
+        lock = builtins.fromTOML (builtins.readFile lockPath);
+      in
+      if builtins.pathExists lockPath && lock.version == 1 then
+        builtins.listToAttrs (
+          map (dep: {
+            name = dep.name or dep.id;
+            value =
+              let
+                path = "${root}/${dep.path or dep.id}@.toml";
+              in
+              if dep ? version && dep ? id then
+                if builtins.pathExists path then
+                  (import ./importAtom.nix) { } path
+                else
```

Comment:
Can add a second case
if there is "${root}/${dep.path or dep.id}/mod.nix" then
So we could potentially skip creating the ...@.toml file?